features/schedule/trigger/feature.ts

```diff
@@ -0,0 +1,63 @@
+import { Feature, retry } from '@temporalio/harness';
+import { ScheduleClient, Connection } from '@temporalio/client';
+import * as assert from 'assert';
+import { randomUUID } from 'crypto';
+
+function delay(ms: number) {
+  return new Promise((resolve) => setTimeout(resolve, ms));
+}
```

Comment:
Can use this API from node 16: https://nodejs.org/api/timers.html#timerspromisessettimeoutdelay-value-options